### PR TITLE
Add a conduktor quick start docker compose

### PR DESCRIPTION
**Pull request by rokh-conduktor**
This compose will be used in the conductor website / documentation

### CODE REVIEW

File: quick-start.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+services:  
+  
+  postgresql:
+    image: postgres:14
+    hostname: postgresql
+    environment:
+      POSTGRES_DB: "conduktor-platform"
+      POSTGRES_USER: "conduktor"
+      POSTGRES_PASSWORD: "some_password"
```
Comment (qboileau) [inline quick-start.yml:11]:
```suggestion
      POSTGRES_PASSWORD: "change_me"
```

**Review by qboileau — COMMENTED** [review]
What about monitoring external storage ?

File: quick-start.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+
+services:  
+  
+  postgresql:
+    image: postgres:14
+    hostname: postgresql
+    environment:
```
Comment (qboileau) [inline quick-start.yml:8]: Missing volume on postgresql 
```
    volumes:
      - pg_data:/var/lib/postgresql/data
```

Comment (rokh-conduktor) [reply quick-start.yml:8]: +1

**Pull request merged**